client/bnpl-announcement/index.js

```diff
@@ -66,32 +62,27 @@ const Dialog = () => {
 			}
 		>
 			<div className="wcpay-bnpl-announcement__payment-icons">
-				<KlarnaPillIcon />
-				<AfterpayClearpayIcon />
-				<AffirmLogoIcon />
+				<img
+					src={ BannerIcon }
+					alt={ __(
+						'Buy now, pay later is here',
+						'woocommerce-payments'
+					) }
+				></img>
 			</div>
 			<h1 id="wcpay-bnpl-announcement">
 				{ __( 'Buy now, pay later is here', 'woocommerce-payments' ) }
 			</h1>
 			<p>
 				{ __(
-					'Boost conversions and give your shoppers additional',
-					'woocommerce-payments'
-				) }
-				<br></br>
-				{ __(
-					'buying power, with buy now, pay later — ',
-					'woocommerce-payments'
-				) }
-				<br></br>
-				{ __(
-					'now available in your WooPayments dashboard *.',
+					// eslint-disable-next-line max-len
+					'Boost conversions and give your shoppers additional buying power, with buy now, pay later — now available in your WooPayments dashboard*.',
```

Comment:
```suggestion
					'Boost conversions and give your shoppers additional buying power, with buy now, pay later — now available in your WooPayments dashboard.*',
```

I know the design has the `*` where you have it, but after some internet searching (and a hunch that it was in the wrong place), I've found a few sources ([1](https://www.thesaurus.com/e/grammar/asterisks/), [2](https://grammarist.com/punctuation/asterisk-symbol-usage-examples/)) that say asterisks should come _after_ punctuation marks:

> When using an asterisk, it is typically considered proper to put the asterisk after every punctuation mark except dashes, in which case the asterisk would come first.

Reply:
Dope, this asterisk has already been living quite a charmed life, with its spacing and position moving around pretty liberally during the life of this PR. I am generally one who is the most offended by improper grammar, so more than happy to make this change and enter into the good graces of good grammar. 😁